SPDY/SPDYSessionManager.m

```diff
@@ -120,8 +123,11 @@ - (void)queueStream:(SPDYStream *)stream
     [_pendingStreams addStream:stream];
     stream.delegate = self;
 
-    NSTimeInterval deferrableInterval = stream.request.SPDYDeferrableInterval;
+    NSTimeInterval fastFailDeferrableInterval = (_fastFailCount > 0) ? ((_lastFastFailTime + kFastFailDeferTime) - CFAbsoluteTimeGetCurrent()) : 0;
+    NSTimeInterval deferrableInterval = fmax(stream.request.SPDYDeferrableInterval, fmin(fastFailDeferrableInterval, kFastFailDeferTime));
```

Comment:
Good to know. My experience with the MIN and MAX macros in a past life is that they'll double evaluate, which is often very bad (though not here), so I just avoid them in general.